recipes/mtspec/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.3.0" %}
+{% set sha256 = "307dc34acefdfae06f055d99d0260cb3529a1fd46d7696be2ab533752aff01a1" %}
+
+package:
+  name: mtspec
+  version: {{ version }}
+
+
+source:
+  fn: mtspec-{{ version }}.zip
+  url: https://files.pythonhosted.org/packages/e8/23/a6b16605dbe1e47a2a6128957ffd4c6d3aa06bd2971de4de3825e1f6f65d/mtspec-0.3.0.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - gcc
+    - numpy
+  run:
+    - python
+    - libgcc
```

Comment:
Can you try using only `libgfortran` instead of `libgcc`? It may not work b/c the default channel does not ship `libquadmath` with `libgfortran` on Linux. If that works we can avoid shipping necessary libs, but if that fails you can do:

```yaml
- libgcc  # [linux]
- libgfortran  # [osx]
```

b/c `conda-forge` has a `libgfortran` for `OS X` that should be enough.